hydra-filters/src/main/java/com/addthis/hydra/data/filter/value/AbstractMatchStringFilter.java

```diff
@@ -258,6 +268,19 @@ public boolean passedFind(String sv) {
         }
     }
 
+    @Override public void postDecode() {
+        if (refreshMinutes == 0) {
+            postDecodeHelper();
+        } else {
+            ScheduledThreadPoolExecutor executor = new ScheduledThreadPoolExecutor(1);
+            // upon JVM termination, wait for the tasks for up to 100ms before exiting the executor service
+            ScheduledExecutorService executorService = MoreExecutors.getExitingScheduledExecutorService(executor, 100, TimeUnit.MILLISECONDS);
+            executorService.scheduleWithFixedDelay(() -> {
```

Comment:
My intellij inspections recommend using a method reference instead of a lambda here: `this::postDecodeHelper`

Reply:
Done